sljit_src/sljitNativeX86_common.c

```diff
@@ -614,11 +631,11 @@ SLJIT_API_FUNC_ATTRIBUTE void* sljit_generate_code(struct sljit_compiler *compil
 		sljit_unaligned_store_sw((void*)(put_label->addr - sizeof(sljit_sw)), (sljit_sw)put_label->label->addr);
 #else
 		if (put_label->flags & PATCH_MD) {
-			SLJIT_ASSERT(put_label->label->addr > HALFWORD_MAX);
+			SLJIT_ASSERT(put_label->label->addr + put_label->label->size > HALFWORD_MAX);
```

Comment:
I think we have discussed that the original assert is correct.

Reply:
without this change the assert will still trigger because the label addr won't be patched to the right value unless it is emitted by the compiler.

so it is correct only if not using hand made labels with big sizes and therefore needs this "tweak" to match the one above that assumed addr will always be updated to some value near code

Reply:
We should not change it because of a test. It is probably better to disable the test if we cannot find a better soultion.